webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -4796,51 +4726,59 @@ msgstr ""
 msgid "landing.interviews-link"
 msgstr "Mehr erfahren"
 
-#: app/templates/eligibility/display_failure.html:7
-#: app/templates/eligibility/display_success.html:7
+#: app/templates/eligibility/display_failure.html:13
+#: app/templates/eligibility/display_success.html:13
 msgid "success_icon"
 msgstr "Abgehakt Icon auf grünem Kreis"
 
-#: app/templates/eligibility/display_failure.html:15
+#: app/templates/eligibility/display_failure.html:22
 msgid "form.eligibility.use-elster"
 msgstr "Wie geht es weiter?"
 
-#: app/templates/eligibility/display_failure.html:16
+#: app/templates/eligibility/display_failure.html:24
 msgid "form.eligibility.use-elster-text"
 msgstr ""
 "Es gibt zwei mögliche Wege Ihre Steuererklärung ohne den Steuerlotsen zu "
 "machen:"
 
-#: app/templates/eligibility/display_failure.html:18
+#: app/templates/eligibility/display_failure.html:26
 msgid "form.eligibility.use-elster.list-item-1"
 msgstr ""
 "Sie können die vollumfänglichen Steuererklärungsvordrucke nutzen. Die "
 "notwendigen Formulare dazu erhalten Sie zum Beispiel in Ihrem Finanzamt."
 
-#: app/templates/eligibility/display_failure.html:35
+#: app/templates/eligibility/display_failure.html:27
 msgid "form.eligibility.use-elster.list-item-2"
 msgstr ""
 "Sie können Ihre Steuererklärung online über das Steuerportal Mein ELSTER "
 "machen."
 
-#: app/templates/eligibility/display_failure.html:37
+#: app/templates/eligibility/display_failure.html:35
 msgid "form.eligibility.use-elster-button"
 msgstr "Zu Mein ELSTER"
 
-#: app/templates/eligibility/display_success.html:15
+#: app/templates/eligibility/display_success.html:22
 msgid "form.eligibility.success-next-step"
 msgstr "Wie geht es weiter?"
 
-#: app/templates/eligibility/display_success.html:16
+#: app/templates/eligibility/display_success.html:24
 msgid "form.eligibility.success-next-step-text"
 msgstr ""
 "Als Nächstes können Sie sich registrieren. Dazu brauchen Sie Ihre Steuer-"
 "Identifikationsnummer und Ihr Geburtsdatum."
 
-#: app/templates/eligibility/display_success.html:27
+#: app/templates/eligibility/display_success.html:36
 msgid "form.eligibility.use-register-button"
 msgstr "Registrieren"
 
+#: app/templates/eligibility/form_marital_status_input.html:6
+msgid "eligibility.marital_status.widowed_note"
+msgstr ""
+"Wenn Sie seit 2020 verwitwet sind und zusammen gelebt haben, bietet Ihnen"
+" der Steuerlotse nur die Möglichkeit, eine gemeinsame Steuererklärung für"
+" Sie beide zu machen (Zusammenveranlagung). Ihre Antworten müssen sich im"
+" Folgenden daher auf Sie beide beziehen."
```

Comment:
I know this is not the correct place. However, I am wondering if I am the only one for whom the text "Sie beide" seems a little strange.

Reply:
Haha, I personally don't ind it but I see your point. :D